src/components/Markings/components/DrawingButtons.js

```diff
@@ -12,25 +12,25 @@ const DrawingButtons = (props) => {
                 disabled={!props.canUndo}
                 type="undo"
                 activated={false}
-                onPress={() => props.onButtonSelected('undo')}
+                onPress={() => props.onUndoButtonSelected()}
```

Comment:
Should 'undo' be removed here, or is this now the default action?

Reply:
Ahh yes, my bad. I didn't initially notice the function changed, only the argument.